build/pkgs/rpds_py/SPKG.rst

```diff
@@ -0,0 +1,18 @@
+rpds_py: Python bindings to Rust's persistent data structures (rpds)
+====================================================================
+
+Description
+-----------
+
+Python bindings to Rust's persistent data structures (rpds)
+
+License
+-------
+
+MIT
+
+Upstream Contact
+----------------
+
+https://pypi.org/project/rpds-py/
+
```

Comment:
It seems that rpds_py replaces pyrsistent. They are standard packages. We have a strict policy regarding changes on standard packages. As I said elsewhere, I think we are making a promise for standard packages that the standard packages are available in the standard install. Replacing a standard package with another silently breaks the promise.

I know that these packages are dependencies of a standard package. So I guess you apply the policy loosely to these. I understand this. So I think we should make the distinction between the main standard packages and the auxiliary standard packages. As there is no need to make the distinction on the technical side, I suggest to add a paragraph (perhaps under "Description") explaining how the package is used in sage and to which standard package it is a dependency and the like, hinting that there may be changes to this package without notice (on sage-devel).

I think these notices should be added to all auxiliary standard packages (later).

Reply:
Yes, that's a good idea and could be a good step towards
- #37586

Reply:
I'll offer that to some extent, the tags `math` and `front-end` may be filling this role, as they are packages that we specifically advertise to users.

Reply:
Thanks. Good enough.